packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -240,224 +234,6 @@ export interface IContextualMenuProps extends IBaseProps<IContextualMenu>, IWith
   hidden?: boolean;
 }
 
-export interface IContextualMenuItem {
-  /**
-   * Optional callback to access the IContextualMenuRenderItem interface. This will get passed down to ContextualMenuItem.
-   */
-  componentRef?: IRefObject<IContextualMenuRenderItem>;
-
-  /**
-   * Unique id to identify the item
-   */
-  key: string;
-
-  /**
-   * Text description for the menu item to display
-   */
-  text?: string;
-
-  /**
-   * Seconday description for the menu item to display
-   */
-  secondaryText?: string;
-
-  itemType?: ContextualMenuItemType;
-
-  /**
-   * Props that go to the IconComponent
-   */
-  iconProps?: IIconProps;
-
-  /**
-   * Custom render function for the menu item icon
-   */
-  onRenderIcon?: IRenderFunction<IContextualMenuItemProps>;
-
-  /**
-   * Props that go to the IconComponent used for the chevron.
-   */
-  submenuIconProps?: IIconProps;
-
-  /**
-   * Whether the menu item is disabled
-   * @defaultvalue false
-   */
-  disabled?: boolean;
-
-  /**
-   * If the menu item is a split button, this prop disables purely the primary action of the button.
-   * @defaultvalue false
-   */
-  primaryDisabled?: boolean;
-
-  /**
-   * [TODO] Not Yet Implemented
-   */
-  shortCut?: string;
-
-  /**
-   * Whether or not this menu item can be checked
-   * @defaultvalue false
-   */
-  canCheck?: boolean;
-
-  /**
-   * Whether or not this menu item is currently checked.
-   * @defaultvalue false
-   */
-  checked?: boolean;
-
-  /**
-   * Whether or not this menu item is a splitButton.
-   * @defaultvalue false
-   */
-  split?: boolean;
-
-  /**
-   * Any custom data the developer wishes to associate with the menu item.
-   */
-  data?: any;
-
-  /**
-   * Callback issued when the menu item is invoked. If ev.preventDefault() is called in onClick, click will not close menu.
-   * Returning true will dismiss the menu even if ev.preventDefault() was called.
-   */
-  onClick?: (
-    ev?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>,
-    item?: IContextualMenuItem
-  ) => boolean | void;
-
-  /**
-   * An optional URL to navigate to upon selection
-   */
-  href?: string;
-
-  /**
-   * An optional target when using href
-   */
-  target?: string;
-
-  /**
-   * An optional rel when using href. If target is _blank rel is defaulted to a value to prevent clickjacking.
-   */
-  rel?: string;
-
-  /**
-   * Properties to apply to a submenu to this item.
-   * The ContextualMenu will provide default values for 'target', 'onDismiss', 'isSubMenu',
-   *  'id', 'shouldFocusOnMount', 'directionalHint', 'className', and 'gapSpace', all of which
-   *  can be overridden.
-   */
-  subMenuProps?: IContextualMenuProps;
-
-  /**
-   * Method to provide the classnames to style the individual items inside a menu. Default value is the getItemClassnames func
-   * defined in ContextualMenu.classnames.
-   * @default getItemClassNames
```

Comment:
The comment re: `getItemClassNames` as default was inaccurate. Looks like we haven't been using that since we started to shift towards `mergeStyles`.